cli/src/cli-rpc-ops.c

```diff
@@ -10572,6 +10574,7 @@ cli_to_glusterd(gf_cli_req *req, call_frame_t *frame, fop_cbk_fn_t cbkfn,
     int i = 0;
     const char **words = NULL;
     cli_local_t *local = NULL;
+    struct cli_state *state = frame->this->private;
```

Comment:
We can move the initialization to line 10583 after frame null check to avoid null dereference warning